ci/Dockerfile-ppc64

```diff
@@ -72,9 +74,6 @@ RUN set -e; \
     && sdk install maven $MAVEN_VERSION \
     && sdk install gradle $GRADLE_VERSION \
     && sdk install sbt $SBT_VERSION \
-    && sdk offline enable \
```

Comment:
I don't think sdkman supports ppc64 that well, so the new java with version types feature are unlikely to work well in this architecture.